frontend/apps/reader/modules/readergesture.lua

```diff
@@ -589,7 +589,7 @@ function ReaderGesture:setupGesture(ges, action)
         if self.is_docless then
             overrides = { 'filemanager_tap' }
         else
-            overrides = { 'readerfooter_tap', }
+            overrides = { 'readerfooter_tap' }
```

Comment:
Or alternatively you could write it like like this:
```
overrides = {
    'readerfooter_tap',
}
```
It makes no difference here, but it would for future diffs.

Reply:
Done, for all of them while I was at it.

Reply:
Huh, never thought of that, will try to remember it next time I deal with a similar construct ;).

Reply:
For short things like the above I don't really mind, but if I notice something like this I'll definitely point out the fact that you could have a diff with one line added vs two lines changed. It just makes it ever so slightly harder to focus on what's relevant. :-)

```lua
{
'adfasd',
';lkj' -- no comma
}
```